docs/workflow/testing/libraries/testing.md

```diff
@@ -1,47 +1,62 @@
 # Testing Libraries
 
-We use the OSS testing framework [xunit](https://github.com/xunit/xunit).
+## Pre-requisites
+Before tests can run the following steps must happen:
+1. Build a runtime (either Mono or coreclr)
+2. Build the libraries
+3. Testhost deployment: Binaries must be arranged into a directory layout that tools/tests expect
```

Comment:
Please remove this step as it should be transparent to the user. No action needs to be taken for this to happen.

Reply:
Leaving this step out entirely feels misleading. The [other docs page](https://github.com/dotnet/runtime/blob/8b116050be5735be8c94dcb8b0cd7554cab1e8dc/docs/workflow/building/libraries/README.md#iterating-on-systemprivatecorelib-changes) shows that explicit awareness and action is required in some some workflows:

> When changing System.Private.CoreLib after a full build, in order to test against those changes, you will need an updated System.Private.CoreLib in the testhost. In order to achieve that, you can build the libs.pretest subset which does testhost setup including copying over System.Private.CoreLib.

Lower on this page there is also explicit mention of running tests on individual assemblies which suggests that people reading this page are likely to be doing the single assembly build workflows where handling the deployment step manually is required.

I could probably restructure this and explain that it happens automatically when building all libs, but it does not happen automatically when rebuilding a single lib. Alternately we could divide up the content some other way?